src/config/dynamic-theme-fixes.config

```diff
@@ -1514,6 +1514,20 @@ img[alt="Bankier.TV"]
 
 ================================
 
+bankofamerica.com
+
+CSS
+.fsdnav-sub-nav-left {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.ad-acct-layout,.ad-acct-detail-card-layout,
+.thrfwl-body,
+.ad-summary-container,.olb-summary-widget-container {
```

Comment:
Woops, missed this. `ad` are those advertisements? While I do support and recommend using adblock we shouldn't be the one removing it.

Reply:
I was worried too, but even with no adblocker active they do not seem to contain any ad.
For example, these are `ad-summary-container` and `.olb-summary-widget-container`:

**No Darkreader**
<img width="1106" alt="No Darkreader" src="https://user-images.githubusercontent.com/29781022/134785346-198659f0-317b-4531-a309-7ab0e3abad55.png">

**Darkreader**
<img width="1106" alt="Darkreader" src="https://user-images.githubusercontent.com/29781022/134785357-ca499897-091f-4cce-8cef-fd54a7afa5be.png">

**Darkreader fixed**
<img width="1106" alt="Darkreader fixed" src="https://user-images.githubusercontent.com/29781022/134785363-1a69de65-d84c-40a5-99ec-7025da987d1a.png">